packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_splash/views/splash_page.dart

```diff
@@ -0,0 +1,36 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+
+import '../../app_extensions.dart';
+import '../../base/models/errors/error_model.dart';
+import '../blocs/splash_bloc.dart';
+
+class SplashPage extends StatelessWidget {
+  const SplashPage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+    body: RxBlocBuilder<SplashBlocType, ErrorModel>(
+      state: (bloc) => bloc.states.errors,
+      builder: (state, snapshot, bloc) =>
+      snapshot.hasData && snapshot.data != null
```

Comment:
Please extract this closure as a private method

Reply:
Done

Reply:
I mean the closure of the builder parameter, not the entire widget 
Check this video for more details: https://www.youtube.com/watch?v=IOyq-eTRhvo&t=2s

Reply:
reworked